cpp/cuopt_remote_server.cpp

```diff
@@ -0,0 +1,457 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
```

Comment:
You may want to refer to this latest PR on adding new shortened licensed header. https://github.com/NVIDIA/cuopt/pull/573/files